### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by chriscar**
Added my 360-degree photosphere viewer example link to scenes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -99,6 +99,7 @@ Use React with A-Frame. Share your A-Frame React Components!
 - [threeschwifty](https://ngokevin.github.com/threeschwifty) &mdash; [Code](https://github.com/ngokevin/threeschwifty) - 360-degree photo and video viewer.
 - [VR GIF](http://x627.com/vr-gif/) &mdash; [Post](http://x627.com/a-frame-is-awesome/) - GIFs in VR.
 - [Virtual Art School Tour](http://stampswebdesign.com/withersc/hud/index.html) (by [@withersc]) - 360-degree photos with navigation.
+- [360-degree photosphere viewer example](https://github.com/IdeaSpaceVR/aframe-360-degree-photosphere)
```
Comment (ngokevin) [inline README.md]: We started to follow the general guidelines of sindresorhus/awesome. Can you check out the [contributing guidelines](https://github.com/aframevr/awesome-aframe/blob/master/contributing.md#adding-to-this-list) and modify things such as Title Casing and Description. Thanks!

**Comment by chriscar** [comment]
I updated the line, hope it's ok! Thanks!

File: README.md [hunk 2]
```diff
@@ -99,6 +99,7 @@ Use React with A-Frame. Share your A-Frame React Components!
 - [threeschwifty](https://ngokevin.github.com/threeschwifty) &mdash; [Code](https://github.com/ngokevin/threeschwifty) - 360-degree photo and video viewer.
 - [VR GIF](http://x627.com/vr-gif/) &mdash; [Post](http://x627.com/a-frame-is-awesome/) - GIFs in VR.
 - [Virtual Art School Tour](http://stampswebdesign.com/withersc/hud/index.html) (by [@withersc]) - 360-degree photos with navigation.
+- [360-Degree Photosphere Viewer Example](https://github.com/IdeaSpaceVR/aframe-360-degree-photosphere) - simple 360-degree viewer for equirectangular photos with a navigation.
```
Comment (cvan) [inline README.md]: Can you link to [your example](https://www.ideaspacevr.org/examples/aframe/aframe-360-degree-photosphere/) and then link to the code by doing `&mdash; [Code](https://github.com/IdeaSpaceVR/aframe-360-degree-photosphere)` (like [this one](https://github.com/aframevr/awesome-aframe/pull/25/files#diff-04c6e90faac2675aa89e2176d2eec7d8R9) above)?

Comment (cvan) [inline README.md]: you can also remove the word "Example" from the title

Comment (cvan) [inline README.md]: also, can you change `with a navigation` to `with navigation`

**Comment by chriscar** [comment]
I updated the line. Thanks!

**Comment by ngokevin** [comment]
Thanks, I'll merge this in and address some formatting things @cvan will introduce.

**Pull request merged**